File: 7KYU/Is n divisible by (...) ?/solution.cs
using System;

namespace Codewars {
  public static class KataChallenge {
    public static bool IsDivisible(params int[] args)
    {
       int fst = args[0];
      
       for(int i = 1; i < args.Length; i++) {
         if(fst % args[i] != 0) {
           return false;
         }
       }
       return true;
    }
  }
}

File: 7KYU/Is n divisible by (...) ?/solution.java
public class Divisible {
    public static boolean isDivisible(int...args) {
        int n = args[0];
      
        for(int i = 0; i < args.length; i++) {
          if(n % args[i] != 0) {
             return false;
          }
        }
        return true;          
    }
}
